sample/src/main/res/layout/fragment_radio_buttons.xml

```diff
@@ -0,0 +1,94 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.core.widget.NestedScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <com.decathlon.android.radio.VitaminRadioButton
+            android:id="@+id/selectedRadioButton"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:checked="true"
+            android:text="Selected by default"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toTopOf="parent"
+            tools:ignore="HardcodedText" />
+
+        <com.decathlon.android.radio.VitaminRadioButton
+            android:id="@+id/unselectedRadioButton"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:checked="false"
+            android:text="Unselected by default"
+            app:layout_constraintStart_toStartOf="parent"
+            app:layout_constraintTop_toBottomOf="@id/selectedRadioButton"
+            tools:ignore="HardcodedText" />
+
+        <com.decathlon.android.radio.VitaminRadioButton
+            android:id="@+id/disabledUncheckedRadioButton"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:checked="true"
+            android:enabled="false"
+            android:text="Disabled by default"
```

Comment:
`by default` seems not necessary here as you cannot change the state in the sample.
`Disabled` or `Disabled checked` should be enough, what do you think ?

Reply:
I'll put `Disabled checked` and `Disabled unchecked` in order to provide both disabled visuals. But indeed `by default` makes no sense since the state is locked.